src/App.tsx

```diff
@@ -1,14 +1,42 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
```

Comment:
the fix is described in the error - just do it
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/093156a2-7e44-4e1b-8078-0985125ab461)

Reply:
Not fixed
![2023-10-06_16h03_26](https://github.com/mate-academy/react_autocomplete/assets/61904995/8b96e1cb-5cd5-41a0-91dd-83a757c95439)